wagtail/admin/templates/wagtailadmin/shared/editing_sessions/module.html

```diff
@@ -36,7 +36,7 @@
     <input type="checkbox" name="is_editing" value="1" data-w-session-target="unsavedChanges" hidden />
     {% if revision_id %}
         {# The ID of the latest revision that was known when this module was initially loaded #}
-        <input type="hidden" name="revision_id" value="{{ revision_id }}" />
+        <input type="hidden" name="revision_id" value="{{ revision_id|unlocalize }}" />
```

Comment:
I think we're better off casting the ID to string in the Python code like we did for the interval:

https://github.com/wagtail/wagtail/blob/a59a2d0585f9b2ee4ff94152a5de4c66b02819df/wagtail/admin/ui/editing_sessions.py#L33-L35

The `|unlocalize` in the template for the ping interval was mistakenly added during merging because we thought we haven't casted it to string. We should remove that.

Also, could you add a test for this? I think you should be able to add another method that calls an existing test method, but with `override_settings` in https://github.com/wagtail/wagtail/blob/a59a2d0585f9b2ee4ff94152a5de4c66b02819df/wagtail/admin/tests/test_editing_sessions.py#L1336 e.g.

```py
@override_settings(USE_THOUSAND_SEPARATOR=True)
def test_edit_view_with_thousand_separator(self):
    # assertRevisionInput will fail if the revision_id is missing |unlocalize
	self.test_edit_view_with_default_interval()
```

Since this has bitten us a few times before – I'm starting to think we really should update our test matrix to have at least one with `USE_THOUSAND_SEPARATOR = True`...

Reply:
Test added. As for

> Alternatively, we might be able to run the entire test suite with USE_THOUSAND_SEPARATOR = True and see if there are already failing tests for them.

The thing is we must set a context a large numbers every time so I don't think adding a setting in the matrix would be enough, extra care for template variable usage from reviewers should go a long way.